src/Airport.js

```diff
@@ -0,0 +1,43 @@
+const Airplane = require(`./Airplane`);
+
+class Airport {
+
+    constructor() {
+    }
+
+    capacity = 10;
+    airportPlanes = [];
+    allLandedPlanes = [];
+
+
```

Comment:
In reference to code just below: Arguably, for clarity reasons it might have been better to break this single conditional statement that is made up of multiple conditions into a few different conditional statements because at the moment the print output doesn't provide the user with precise information but rather a list of possible reasons for failure. On the other hand, however, you could argue that this is what the testing is designed to clarify... Perhaps it's confusing assigning a numerical value to the weather: It would almost certainly have been clearer to have a storm variable as a Boolean instead (either there is or isn't a storm).